timestamp-client/src/test/java/com/palantir/timestamp/RateLimitedTimestampServiceTest.java

```diff
@@ -30,27 +29,24 @@
 public class RateLimitedTimestampServiceTest {
```

Comment:
How confident are we in these tests? The `RateLimitedTimestampService` class is a little wonky - but I guess it's been proven in the field so no sense in re-implementing just yet.